scipy/spatial/transform/rotation.py

```diff
@@ -657,3 +658,32 @@ def __mul__(self, other):
         if self._single and other._single:
             result = result[0]
         return self.__class__(result, normalized=True)
+
+    def apply(self, points):
```

Comment:
I'm not sure if I like name `points` here. The conceptual problem with it is that rotation of points makes sense around some origin and we might want also apply translation to points. That is the position of "points" relative to the frame origin is important.

But since this class represents only rotations, we are better to apply the rotation to "free vectors", that is which starting point is irrelevant.

So I suggest to use either `v` or `vectors`.